ci/scenario_helpers/threading_handlers.py

```diff
@@ -185,12 +185,13 @@ def poll_io(cls, r_semaphore, required_thread_amount, shared_resource, downed_ti
                 continue
             # Check if any errors occurred - possible due to the nature of the write data with screens
             # If the fio has had an error, it will break and output to the output file
-            errors = {}
+            errors = set()
```

Comment:
shouldn't this be a list?
as you want sth ordered but not throw away duplicates right?